core/src/main/java/org/elasticsearch/indices/IndexingMemoryController.java

```diff
@@ -52,7 +52,8 @@
 public class IndexingMemoryController extends AbstractComponent implements IndexingOperationListener, Closeable {
 
     /** How much heap (% or bytes) we will share across all actively indexing shards on this node (default: 10%). */
-    public static final Setting<ByteSizeValue> INDEX_BUFFER_SIZE_SETTING = Setting.byteSizeSetting("indices.memory.index_buffer_size", "10%", Property.NodeScope);
+    public static final Setting<ByteSizeValue> INDEX_BUFFER_SIZE_SETTING = Setting.memorySizeSetting("indices.memory.index_buffer_size",
+            "10%", Property.NodeScope);
```

Comment:
Nit: Can we reformat this by just moving everything after the equals sign to the next line with continuation indentation so that all the parameters and the `Setting.memorySizeSetting` invocation should fit on one line then?